Sources/Plot/API/HTMLElements.swift

```diff
@@ -87,6 +87,15 @@ public extension Node where Context: HTML.BodyContext {
         .element(named: "a", nodes: nodes)
     }
 
+    /// Add an `<a>` HTML element within the current context.
+    /// - Parameters:
+    ///   - href: The URL to assign.
+    ///   - nodes: The element's attributes and child elements.
+    static func a(href url: URLRepresentable, _ nodes: Node<HTML.BodyContext>...) -> Node {
+        .element(named: "a",
+                 nodes: [.attribute(named: "href", value: url.string)] + nodes)
+    }
```

Comment:
This is safe against double `href` attributes, because `nodes` cannot actually contain a `.href(...)`-created attribute (due to `AnchorContext` requirement).

Of course, someone could add `.element(named: "href", ...)` but that's fundamentally a problem with the escape hatch, isn't it.